examples/oneapi/dpc/source/example_util/dpc_helpers.hpp

```diff
@@ -19,6 +19,21 @@
 #include <vector>
 #include <sycl/sycl.hpp>
 
+#include "oneapi/dal/array.hpp"
+#include "oneapi/dal/detail/array_utils.hpp"
+
+template <typename Type>
+inline auto to_host(const oneapi::dal::array<Type>& array) {
+    oneapi::dal::detail::default_host_policy policy{};
```

Comment:
add 
#include "oneapi/dal/detail/policy.hpp"
or array.hpp
to resolve fail in CI